table_helpers.go

```diff
@@ -184,7 +184,13 @@ func (t Table) InsertQueryList(r *rand.Rand, valueOverrides []map[string]interfa
 	return buf.String(), args
 }
 
+const defaultLoadConcurrency = 16
```

Comment:
why even bother with a const? it's not like this value is used elsewhere? inlining the code will make the code more linear to read?